L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc

```diff
@@ -90,10 +90,26 @@ const bool l1t::AXOL1TLCondition::evaluateCondition(const int bxEval) const {
   int useBx = bxEval + m_gtAXOL1TLTemplate->condRelativeBx();
 
   //HLS4ML stuff
-  std::string AXOL1TLmodelversion = m_AXOL1TLmodelversion;  //config loading method
+  // std::string AXOL1TLmodelversion = m_AXOL1TLmodelversion;  //loading from menu
+  std::string AXOL1TLmodelversion = "GTADModel_" + m_gtAXOL1TLTemplate->modelVersion();  //loading from menu/template
```

Comment:
Why is it necessary to hardcode `GTADModel_` for the AXO models ? Couldn't the L1T-menu xml specify `GTADModel_vN` rather than just `vN` ?

Reply:
I think that is a question to @caruta and the uGT team @herbberg and @arnobaer 
I imagine we can do it either way, but just need to have consistent name changes.

Reply:
I guess it might be related to this convention by the uGT team for the FW versioning vs the HLS4ML version names: 
https://globaltrigger.web.cern.ch/upgrade/tme/models vs https://github.com/cms-hls4ml/AXOL1TL/tree/main/AXOL1TL_v3

fyi @quinnanm  @jmduarte @thesps